api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -369,22 +383,31 @@ public void setVersion(int v) {
     }
 
     /*
-     * Tests a string and returns true if the string counts as a reserved token in the Java language.
+     * Tests a string and returns true if the string contains a reserved token for the Set-Cookie header.
      * 
```

Comment:
The use of token here is confusing as token has a specific meaning in the HTTP spec. This method is checking for invalid characters, not tokens.

Reply:
Please propose. Original javadoc also didn't make any sense wrt actual impl.

Reply:
hows that?